tests/BasicSearchTests.FunctionalTests.Core/AutocompleteFunctionalTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using BasicSearchTests.FunctionalTests.Core.Models;
+using BasicSearchTests.FunctionalTests.Core.TestSupport;
+using Xunit;
+using System.Net;
+
+namespace BasicSearchTests.FunctionalTests.Core
+{
+    public class AutocompleteFunctionalTests
```

Comment:
I do not see this "convention" being used anywhere, the only functional tests I am aware of are in Gallery and they don't use it.